packages/react-storage/src/components/StorageBrowser/Views/LocationActionView/UploadControls.tsx

```diff
@@ -29,13 +29,18 @@ const { Cancel, Exit, Primary, Summary, Table } = Controls;
 
 interface LocationActionViewColumns extends CancelableTask {
   type: string;
+  folder: string;
```

Comment:
When iterating over the rows, can the folder name be read from `CancelableTask`, e.g. `task.data.webkitRelativePath`?

Reply:
it can be, but this property is to help set up the columns in the old table 😔. I do access `task.data.webkitRelativePath` to actually get the value though 

https://github.com/aws-amplify/amplify-ui/blob/77dd7259a04525d577cd48ce89edc64ff77ebc70/packages/react-storage/src/components/StorageBrowser/Views/LocationActionView/UploadControls.tsx#L214-L222

Reply:
Got it, works for now